CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Requests with empty body and application/json Content-Type Header will now
   return 400 error instead of 500 error.
   [cyberark/conjur#1968](https://github.com/cyberark/conjur/issues/1968)
+- Conjur now audit Set secret events with the update operation.Previously, the operation for this event was empty.
```

Comment:
do we have an issue for this? can we add it here o align with the CHANGELOG style?

Reply:
I am still waiting for the issue number

Reply:
i would change the text to `Conjur now properly sets the 'operation' field of a secret update audit message`

Reply:
Text changed